tests/actionExecutor.spec.ts

```diff
@@ -120,6 +120,41 @@ describe("Action executor tests: ", () => {
 
     done();
   });
+
+  it("should retry action twice", async done => {
```

Comment:
🙃 Great that there is a test for this now. Could we add another one that tests that we are not running into an endless loop of retries if the comit-node is actually down? (`it fails when retrying more often than max retries` or similar)